### DISCUSSION TITLE
Licence

### DISCUSSION THREAD

**Discussion by cryptonitiche** [BitMaker-hub/NerdMiner_v2#541] [Q&A]
After the update, the license flew off. Where can I get it? I bought a miner on Marketplace. I wrote to the seller, I am waiting for an answer.

**Comment by BitMaker-hub** [comment]
Nerdminer doesn't has a propietary license, and should not have.
It is fully opensource and any work done from this code should also be opensourced

The Heltec fork is not doing it right

**Reply by cryptonitiche** [reply]
Ok, im  update it here: https://flasher.bitronics.store 1.6.3V
It works! 😊
How do I check for updates?
